modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/java/com/liferay/dynamic/data/mapping/type/select/internal/SelectDDMFormFieldValueValidator.java

```diff
@@ -87,9 +87,11 @@ protected JSONArray createJSONArray(String fieldName, String json)
 				_log.debug(jsone, jsone);
 			}
 
-			throw new DDMFormFieldValueValidationException(
```

Comment:
Why are we sending values that are not JSONArray for select types. We should fix this. Please ping @diogo-lins